src/pages/adminActions/AdminActions.jsx

```diff
@@ -23,12 +23,7 @@ export default function AdminActions() {
         style={{ padding: '16px 0 16px 16px' }}
         id="ADMINISTRATIVE_ACTIONS"
       />
-      <ButtonLink
-        href="/admin"
-        style={{ marginTop: 8, width: 'fit-content' }}
-        display="back"
-        id="BACK"
-      />
+      <SettingsBreadcrumbs currentPageTextId="ADMINISTRATIVE_ACTIONS" />
```

Comment:
Presumably, we will be wanting breadcrumbs for more than just this path? I'm guessing that this is just you establishing the pattern for me and Emily?

Reply:
I made this change in all of the control panel pages! So hopefully it's all done now :)